.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Extractors/DutchSetExtractorConfiguration.cs

```diff
@@ -60,7 +63,7 @@ public DutchSetExtractorConfiguration(IDateTimeOptionsConfiguration config)
 
         public IDateTimeExtractor DateTimePeriodExtractor { get; }
 
-        bool ISetExtractorConfiguration.CheckBothBeforeAfter => DateTimeDefinitions.CheckBothBeforeAfter;
+        bool ISetExtractorConfiguration.CheckBothBeforeAfter => true;
```

Comment:
Why not read the config property?

Reply:
CheckBothBeforeAfter is false in the YAML, we need to set it to true only in SetExtractor.

Reply:
Not sure if other cases require it as well, but if this is the case for now, would be good to have a comment here + in the YAML config. Otherwise it will become very messy in the future when we try to use the flag in YAML.

Reply:
ok, comment added.